README.md

```diff
@@ -2,6 +2,8 @@
 
 [Rust RFCs]: #rust-rfcs
 
+This book is a collection of all RFCs that have been accepted.
+
 The "RFC" (request for comments) process is intended to provide a consistent
 and controlled path for changes to Rust (such as new features) so that all
 stakeholders can be confident about the direction of the project.
```

Comment:
Probably the bit being added would make more sense as the second paragraph, after the nature of an RFC has been introduced.

Reply:
No, I don't think that is right. The very first thing this book should say is what it is about at the highest level possible. Then the details of that thing can be explained later.

Also to be able to understand """The "RFC" process ... the direction of the project.""" it helps greatly if you already know what this book is about.